### PR TITLE
Fix fbpspec

### LINKED ISSUE #587: fbp-spec support not working
Was disabled while porting to Webpack (#585). Also had some issues before this, so should be verified working against some runtime.

Partially related to #561, as that is area where tests status is

### PR DESCRIPTION
**Pull request by jonnor**
Fixes #587

### CODE REVIEW

**Comment by jonnor** [comment]
Example for a failing tests, 
flowhub-fbpspec-failingtest

**Comment by jonnor** [comment]
No idea why VersionEye now complains, nothing has changed wrt to license of fbp-spec...

**Comment by bergie** [comment]
@jonnor VersionEye has this bug where occasionally they fail when they see a new dependency they haven't fetched yet

File: package.json [hunk 1]
```diff
@@ -67,12 +67,14 @@
...
     "mocha": "^3.2.0",
+    "simple-server": "^1.0.1",
     "sinon": "^1.17.7",
     "syn": "0.2.2",
     "yaml-include-loader": "^1.0.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt test",
+    "start": "./node_modules/.bin/simple-server . 9999"
```
Comment (bergie) [inline package.json]: This is a convenient change. Should be mentioned in the README

**Comment by bergie** [comment]
@jonnor please mention `npm start` in README and the bug fix in CHANGES

**Pull request merged**